packages/alsa-plugins/build.sh

```diff
@@ -0,0 +1,41 @@
+TERMUX_PKG_HOMEPAGE=https://www.alsa-project.org
+TERMUX_PKG_DESCRIPTION="The Advanced Linux Sound Architecture (ALSA) - plugins"
```

Comment:
`alsa-plugins` currently has a build script
https://github.com/termux/termux-packages/blob/ce597343cc65398a7b00e3237333af0d3296a2af/disabled-packages/alsa-plugins/build.sh#L1-L13
and config file
https://github.com/termux/termux-packages/blob/ce597343cc65398a7b00e3237333af0d3296a2af/disabled-packages/alsa-plugins/asound.conf#L1-L16
in the `disabled-packages` section of the repository.

It may or may not be useful to have a look at that to see if anything from that still applies to this new version.

(The version in `disabled-packages` should also subsequently be removed, though that can be done in a follow up commit.)

Reply:
Oh, how silly of me to have missed that! I'll have a look at the disabled package and update this branch.

Reply:
Considering it hasn't really been touched in 7 years it's a completely reasonable thing to have missed.

I think the main thing of note in the earlier version of the package build is that it's completely permissible to have the config file as a separate file inside the package directory.

I would even somewhat prefer this option over a heredoc personally since it rules out any mishaps due to heredoc semantics shenanigans.

Reply:
What's the preferred way of including a small static file? `disabled-packages/alsa-plugins` puts it into the package directory next to `build.sh`, while I've used a heredoc to create it at build time.

Reply:
We don't enforce one method over another.
It is mostly up to the discretion of the packager.

Heredocs are great if you need to include dynamic information in a config file.
Which I don't think is the case here.

A separate standalone file in the package directory has the benefit of being very obviously there.
So if changes need to be made to the config file they can be made in the standalone file and a package revision can be added in the build script.
I guess what I'm trying to get at is a standalone config file disentangles the configuration file from the build script of the package.